00-open-a-shapefile.Rmd

```diff
@@ -78,237 +75,254 @@ Working With Shapefile Attributes in R </a>
-vector that we are working with (point, line or polygon) as follows: 
-
-* **Points:** In a point vector dataset, each point is represented by a single x,y
-coordinate location. There can be many points in a vector point file. Examples:
-plot locations or the locations of individual trees.
-* **Lines:** Lines are composed of many points that are connected. For instance,
-a road or a stream is a series of straight line segments, each bend in the road
-or stream represents a vertex that has an x,y location associated with it. A
-line consists of AT LEAST 2 vertices that are connnected.
+Vector data are composed of discrete geometric locations (x,y values) that
+define object vertices. The organization of the geometric locations, determines
+the type of vector that we are working with (point, line or polygon). 
+
+<figure>
+    <a href="http://geographerswasaut.blogspot.com/2015/01/the-geographical-information-system.html">
+    <img src="http://4.bp.blogspot.com/-e2352F7HlwM/VKa8xr1ZPcI/AAAAAAAAAXk/ETV0gQVF9dc/s1600/4.png"></a>
+    <figcaption>Vector data can be a point, line, or polygon.  Image Source: E. Chrisant.</figcaption>
+</figure>
+
+* **Points:** Each point, a vertex, is represented by a single x,y coordinate
+location. There can be many points in a vector point file. Examples: sampling
+locations or the location of individual trees.
+* **Lines:** Lines are composed of many (at least 2) vertices, or points, that
+are connected. For instance, a road or a stream is a series of straight line
+segments, each bend in the road or stream represents a vertex that has an x,y
+location associated with it.
 * **Polygons:** A polygon consists of 3 or more vertices that are connected and 
-closed. Thus the outlines of plot boundaries, lakes, oceans, states or 
-countries are often represented by polygons. Ocassionally, a 
-polygon can have a hole in the middle of it (like a donut), this is something to
-be aware of but not an issue we will deal with further.
-
-NOTE: a line can represent a boundary as well - however a line will not create a 
-closed object.
-
-**INSERT GRAPHIC HERE**
+closed. Thus the outlines of plot boundaries, lakes, oceans, and states or 
+countries are often represented by polygons. Ocassionally, a polygon can have a
+hole in the middle of it (like a donut), this is something to be aware of but
+not an issue we will deal with further.
+
+NOTE: a line can represent a boundary as well - however, a line will not create
+a closed object.
+
+<figure>
+    <a href="http://www.geography.hunter.cuny.edu/~jochen/GTECH361/lectures/lecture05/concepts/03%20-%20Geographic%20data%20models.html">
+    <img src="http://www.geography.hunter.cuny.edu/~jochen/GTECH361/lectures/lecture05/concepts/03%20-%20Geographic%20data%20models_files/image001.gif"></a>
+    <figcaption>Each vertices is an x,y coordinate. Examples of a point (a
```

Comment:
eeks - we can do better! graphic quality is important. talk to colin about what he's created! my favorite graphics to use are the ESRI help ones. but otherwise we can make ones that are clean and professional.